packages/contracts-bedrock/src/L2/SuperchainWETH.sol

```diff
@@ -91,4 +91,9 @@ contract SuperchainWETH is WETH98, ICrosschainERC20, ISemver {
 
         emit CrosschainBurn(_from, _amount);
     }
+
+    /// @inheritdoc IERC165
+    function supportsInterface(bytes4 _interfaceId) public view virtual returns (bool) {
```

Comment:
While we're at it, I don't see any harm in adding in all the other interfaces we support in this contract. e.g. IERC20 etc...